### ISSUE TITLE
AWS - Preexisting ECS task definitions marked as "INACTIVE"

### ISSUE THREAD

**Issue by gregwchase** [dask/dask-cloudprovider#241]
**What happened**:

When importing a CSV, an error was thrown due to a local file not being in the current directory. After this error is thrown, the following events occur:
* The Dask scheduler fails to reconnect
* All scheduler and worker nodes are removed unexpectedly
* All preexisting ECS task definitions are marked `INACTIVE`

**What you expected to happen**:

* A `FileNotFoundError`, the cluster remains online, and preexisting ECS task definitions remain in the `ACTIVE` state.

**Minimal Complete Verifiable Example**:

```python
from dask_cloudprovider.aws import FargateCluster
from dask.distributed import Client
import dask.dataframe as dd

cluster = FargateCluster(n_workers=2,
                         tags={"Name": "dask-dev", "Environment": "test"}
                        )

client = Client(cluster)

X_train = dd.read_csv("../X_train_data.csv")
```

The following error is produced when Dask can't find the CSV file. Shortly afterwards, another error is thrown regarding reconnecting to the scheduler. At this point, the scheduler and workers are removed, and any preexisting task definitions are marked as `INACTIVE`.

```python
FileNotFoundError: [Errno 2] No such file or directory: '/Users/gregchase/Desktop/data/X_train_data.csv'

distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client
_GatheringFuture exception was never retrieved
future: <_GatheringFuture finished exception=CancelledError()>
asyncio.exceptions.CancelledError

```

**Anything else we need to know?**:

**Environment**:

- Dask version: 2021.01.1
- Python version: 3.8.3
- Operating System: macOS Catalina 10.15.6
- Install method (conda, pip, source): pip

**Comment by jacobtomlinson** [comment]
Thanks for raising this.

The task definition issue was fixed in #238 and will be in the next release.

You appear to be trying to access data on your local machine from your Fargate Dask workers. This will not be possible.

A common workflow is to place the file on S3 and then access it directly from there.

**Comment by gregwchase** [comment]
Thanks @jacobtomlinson - issue looks to be resolved in the latest release, as you mentioned above. Closing this issue since it's resolved.
